src/templates/prepared-map/map-templates/http.ts

```diff
@@ -0,0 +1,38 @@
+// TODO: This can be divided into more partials
+export default `// 1) Change HTTP method and path to make an HTTP call
+http POST "/endpoint" {
+
+  // 2) Specify security identifier from Provider JSON definition
```

Comment:
I'm so sorry for being that kind of guy who invents something and then criticizes the implementation 🙈 😄 ...

...but I think we should programmatically increment the step numbers since `2)` is optional, and so is `3)` maybe?)

I only realized this today when revisiting the suggestion in Slack 🤔